lib/src/main/java/com/orange/ods/compose/component/menu/OdsDropdownMenu.kt

```diff
@@ -0,0 +1,86 @@
+/*
+ *
+ *  Copyright 2021 Orange
+ *
+ *  Use of this source code is governed by an MIT-style
+ *  license that can be found in the LICENSE file or at
+ *  https://opensource.org/licenses/MIT.
+ * /
+ */
+
+package com.orange.ods.compose.component.menu
+
+import androidx.compose.foundation.background
+import androidx.compose.material.DropdownMenu
+import androidx.compose.material.DropdownMenuItem
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.DpOffset
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.window.PopupProperties
+import com.orange.ods.compose.theme.OdsTheme
+
+/**
+ * <a href="https://system.design.orange.com/0c1af118d/p/07a69b-menus/b/862cbb" class="external" target="_blank">ODS menus</a>.
+ *
+ * @see androidx.compose.material.DropdownMenu
+ *
+ * @param expanded Whether the menu is currently open and visible to the user
+ * @param onDismissRequest Called when the user requests to dismiss the menu, such as by
+ * tapping outside the menu's bounds
+ * @param offset [DpOffset] to be added to the position of the menu
+ */
+@Composable
+fun OdsDropdownMenu(
+    expanded: Boolean,
+    onDismissRequest: () -> Unit,
+    modifier: Modifier = Modifier,
+    offset: DpOffset = DpOffset(0.dp, 0.dp),
+    properties: PopupProperties = PopupProperties(focusable = true),
+    menuItems: List<OdsMenuItem> = emptyList()
```

Comment:
Another possibility is to replace the `menuItems` parameter with `content: @Composable ColumnScope.() -> Unit` and write:

```kotlin
fun ColumnScope.OdsDropdownMenuItem(
    text: String,
    onClick: () -> Unit,
    enabled: Boolean = true
) {
...
}
```

The advantage is that the ODS API will look like the Material API and the user can still add non-ODS content to the menu (which can be a drawback, it depends on what we want to do at the end).